metadata-ingestion/src/datahub/ingestion/source/mode.py

```diff
@@ -268,7 +285,7 @@ def __init__(self, ctx: PipelineContext, config: ModeConfig):
         except HTTPError as http_error:
             self.report.report_failure(
                 key="mode-session",
-                reason=f"Unable to retrieve user "
+                reason=f"Unable to verify connection"
                 f"{self.config.token} information, "
```

Comment:
not sure we want to log the token

unless this is not an auth token, but something else

Reply:
ahh, correct, I didn't notice we have this. 
We shouldn't log this, you are right.